docs/HISTORY.txt

```diff
@@ -5,6 +5,11 @@ Changelog
 3.2 (unreleased)
 ----------------
 
+- Change/Upgrade default values for image related paths [mathias.leimgruber]
+  - Change location of marker.png according collective.geo.openlayers 4.0
```

Comment:
restructured text requires an empty line obove this line in order to detect the nested list correctly 😉